test/e2e_node/builder/build.go

```diff
@@ -29,6 +29,8 @@ import (
 )
 
 var k8sBinDir = flag.String("k8s-bin-dir", "", "Directory containing k8s kubelet binaries.")
+var useDockerizedBuild = flag.Bool("use-dockerized-build", false, "Use dockerized build for test artifacts")
```

Comment:
FWIW I recommend this as the default in the future for consistency reasons. You need docker available regardless to build images so switching doesn't change requirements.

Reply:
Ack. I can follow up with a separate PR to default dockerized build with more comphrehensive testing.